mdbenchmark/mdengines/utils.py

```diff
@@ -101,20 +101,29 @@ def analyze_benchmark(engine, benchmark):
     threads = np.nan
     hyperthreading = np.nan
     module = None
+    multidir = np.nan
 
     # search all output files
     output_files = glob(
         os.path.join(benchmark.relpath, PARSE_ENGINE[engine.NAME]["analyze"])
     )
     if output_files:
-        with open(output_files[0]) as fh:
-            performance = parse_ns_day(engine, fh)
-            fh.seek(0)
-            ncores = parse_ncores(engine, fh)
+        for f in output_files:
+            performance = []
+            ncores = []
+            with open(f) as fh:
+                performance.append(parse_ns_day(engine, fh))
+                fh.seek(0)
+                ncores.append(parse_ncores(engine, fh))
+        performance = np.mean(performance)
```

Comment:
We would want to form the sum not the average of performance values. Running `-multidir` will most of the time, produce shorter individual trajectories. What the user will be interested in is: can I increase my cumulative performance by running multiple simulations per node?

```suggestion
        performance = np.sum(performance)
```